packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -112,6 +76,17 @@ class InspectorController extends DisposableController
       details = null;
     }
 
+    await serviceManager.onServiceAvailable;
```

Comment:
we should check whether this controller has been disposed after this async gap and bail on the rest of the init in that case.